src/Core/Animation/Skeleton.cpp

```diff
@@ -196,11 +182,11 @@ void Skeleton::getBonePoints( const uint i, Vector3& startOut, Vector3& endOut )
         {
             endOut += m_modelSpace[child].translation();
         }
-        endOut *= ( 1.f / children.size() );
+        endOut *= ( 1_ra / children.size() );
     }
 }
 
-Vector3 Skeleton::projectOnBone( int boneIdx, const Ra::Core::Vector3& pos ) const {
+Vector3 Skeleton::projectOnBone( uint boneIdx, const Ra::Core::Vector3& pos ) const {
```

Comment:
Why not using `Ra::Core::Index` here ?

Reply:
because of the `Ra::Core::AdjacencyList` class which should be rewritten to take that into account, which is out of the scope of this PR.